plugins/modules/packaging/language/gem.py

```diff
@@ -194,10 +205,16 @@ def uninstall(module):
         return
     cmd = get_rubygems_path(module)
     environ = get_rubygems_environ(module)
+    ver = get_rubygems_version(module)
     cmd.append('uninstall')
+    if ver and ver >= (2, 5, 2):
```

Comment:
Should this be appended to every command when version is > 2.5.2, I'm not familiar with `gem`, but this is not what I understand from talks here and documentation.

Reply:
refactoring mistake, fixed